stylesheet.css

```diff
@@ -1,16 +1,25 @@
+.headerImg {
```

Comment:
Is this still important? It looks like we changed the class name for the image itself